model-api/src/commonMain/kotlin/org/modelix/model/api/ITreeChangeVisitor.kt

```diff
@@ -26,6 +26,13 @@ interface ITreeChangeVisitor {
      */
     fun containmentChanged(nodeId: Long)
 
+    /**
+     * Called when the concept of a node has changed.
+     *
+     * @param nodeId id of the affected node
+     */
+    fun conceptChanged(nodeId: Long)
```

Comment:
```suggestion
    fun conceptChanged(nodeId: Long) {}
```

Default implementation required to avoid a breaking change.

Reply:
Done